### PR TITLE
PCT: Adapt test for 2.340 structure changes

### PR DESCRIPTION
**Pull request by timja**
Required for https://github.com/jenkinsci/bom/pull/955 and https://github.com/jenkinsci/bom/pull/960

Can we get a release please?

### CODE REVIEW

File: src/test/java/com/cloudbees/jenkins/support/SupportTestUtils.java [hunk 1]
```diff
@@ -276,7 +277,11 @@ public static void testPermissionToSeeAction(JenkinsRule j,
             HtmlPage page = wc.goTo(baseUrl);
             HtmlDivision sidePanel = (HtmlDivision) page.getElementById("side-panel");
             assertEquals(userPrivileged + " should be able to see the Support action", 1,
-                sidePanel.getElementsByAttribute("a", "title", action.getDisplayName()).size());
```
Comment (timja) [inline src/test/java/com/cloudbees/jenkins/support/SupportTestUtils.java:279]: title attribute is no longer set, just iterate through instead, could switch to xpath too but this is quite straightforward

**Comment by timja** [comment]
Passed in bom https://github.com/jenkinsci/bom/pull/960

**Review by Dohbedoh — APPROVED** [review]

**Comment by Dohbedoh** [comment]
the windows CI failed on `master`, trying again https://ci.jenkins.io/blue/organizations/jenkins/Plugins%2Fsupport-core-plugin/detail/master/265/pipeline

**Comment by Dohbedoh** [comment]
@timja finally went through: https://github.com/jenkinsci/support-core-plugin/releases/tag/1148.vedff8cb_56a_da_

**Comment by timja** [comment]
Thanks!

**Pull request merged**